Packages/com.unity.inputsystem/InputSystem/Controls/InputControlPath.cs

```diff
@@ -721,6 +721,39 @@ public static bool Matches(string expected, InputControl control)
             return MatchesRecursive(ref parser, control);
         }
 
+        internal static bool MatchControlComponent(ref ParsedPathComponent expectedControlComponent, ref InputControlLayout.ControlItem controlItem)
```

Comment:
Since the expectedControlComponent is not modified, but used as an input reference, you can change from a `ref` to an `in` and make the intention more clear. 
```suggestion
        internal static bool MatchControlComponent(in ParsedPathComponent expectedControlComponent, ref InputControlLayout.ControlItem controlItem)
```